src/ActivationManager.cc

```diff
@@ -0,0 +1,199 @@
+#include "zeek/ActivationManager.h"
+
+#include "zeek/Desc.h"
+
+using namespace std;
+
+namespace zeek::detail
+	{
+
+void ActivationEvent::Dump(int indent_level) const
```

Comment:
Conceptually, I guess it does.  However I'd rather wait on doing so when we find that we actually need it. I created the method just to debug whether the events were being correctly characterized; indeed I could just remove it at this point, though it might still prove handy for debugging the C++ script compiler's support for conditionals when I get around to that ...

Reply:
(in particular, would rather wait because it's non-trivial work to convert it)